img2lambda/cli/main.go

```diff
@@ -59,11 +59,13 @@ func createApp() (*cli.App, *types.CmdOptions) {
 		cli.StringFlag{
 			Name:        "description, desc",
 			Usage:       "The description of this layer version",
+			Value:       "created by img2lambda from image <name of the image>",
```

Comment:
I believe this will actually set the value of the description option to this static string.  Can you check that?  If so, you'll need to add this to the Usage string, and not set the Value property.

Same for license flag

Reply:
You're right. I've updated this now.